src/coreclr/vm/perfmap.cpp

```diff
@@ -86,29 +94,121 @@ void PerfMap::Initialize()
         
         s_enabled = true;
     }
+
+    if (sendExisting)
+    {
+        if (type == PerfMapType::ALL || type == PerfMapType::JITDUMP)
```

Comment:
It might be worthwhile to add a comment here that this is iterating through restored R2R methods, and this is only valuable when jitdump is used, as perfmap is for jitted code only.

Reply:
Added a comment